tests/test/libs/helpers/helpers.bash

```diff
@@ -0,0 +1,23 @@
+function Create-AWS-Creds () {
+  mkdir "${1}/.aws"
+  cat << EOF > "${1}/.aws/credentials"
+[${2}]
+aws_access_key_id=XXXXXXXXXXXXXXXXXX
+aws_secret_access_key=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+region=us-east-1
```

Comment:
Can we use here AWS_REGION env var?

Reply:
what is the purpose?

Reply:
I think it impacts nothing, but in this case we will have standard cred profile for tests using